flight/Modules/ManualControl/tablet_control.c

```diff
@@ -97,8 +97,9 @@ int32_t tablet_control_select(bool reset_controller)
 
 			// Command to not move. This code is identical to that in manualcontrol
 			LoiterCommandData loiterCommand;
-			loiterCommand.Forward = 0;
-			loiterCommand.Right = 0;
+			loiterCommand.Pitch = 0;
+			loiterCommand.Roll = 0;
+			loiterCommand.Throttle = 0.5f;
```

Comment:
I'm sure this is safe but could you confirm that to me :) this isn't going to make things shoot up, right?